File: python/rna-transcription/rna_transcription.py
mapping = {
    'G': 'C',
    'C': 'G',
    'T': 'A',
    'A': 'U',
}

def to_rna(dna_strand):
    rna = []
    for i in dna_strand:
        rna.append(mapping[i])
    return ''.join(rna)

File: swift/rna-transcription/Sources/RnaTranscription/RnaTranscription.swift
let mapping : [String:String] = [
    "G": "C",
    "C": "G",
    "T": "A",
    "A": "U",
]

func toRna(_ dna: String) -> String {
    var rna = ""
    let dna = Array(dna)
    for c in dna {
        let c = String(c)
        rna += mapping[c]!
    }
    return rna
}
